tests/login.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require('supertest')
```

Comment:
Redux changes are not required here. Please remove these changes. They are not part of your ticket.

Reply:
removed thanks